cli/lsp/tsc.rs

```diff
@@ -4212,6 +4244,9 @@ impl TscSpecifierMap {
       return specifier.to_string();
     }
     let mut specifier = original.to_string();
+    if !specifier.as_str().contains("/node_modules/@types/node/") {
+      specifier = specifier.replace("/node_modules/", "/$node_modules/");
```

Comment:
I added a comment. For some reason tsc doesn't give completions for these paths. But only when `node_modules/.deno` is used (i.e. when using `deno install` instead of `npm install`). I couldn't figure out why.